boards/arm64/index.rst

```diff
@@ -5,6 +5,9 @@ ARM64 Boards
 
 .. toctree::
    :maxdepth: 1
-   :glob:
 
-   **/*
+   bcm958402m2_a72/doc/index.rst
+   fvp_baser_aemv8r/doc/index.rst
+   nxp_ls1046ardb/doc/index.rst
+   qemu_cortex_a53/doc/index.rst
+   xenvm/doc/index.rst
```

Comment:
I'm not a documentation expert but I wonder if there is a way to avoid listing all the index files here.

Reply:
I tried something like `*/doc/index.rst`, but it doesn't work, gives me this error:
```
Warning, treated as error:
/workdir/zephyr/doc/_build/rst/doc/boards/arm64/index.rst:6:toctree glob pattern '*/doc/index.rst' didn't match any documents
ninja: build stopped: subcommand failed.
```
I'm also curious if there's a better way.

Reply:
`**/index.rst` is not working, got similar error:
```
Warning, treated as error:
/workdir/zephyr/doc/_build/rst/doc/boards/arm64/index.rst:6:toctree glob pattern '**/index.rst' didn't match any documents
ninja: build stopped: subcommand failed.
```

I also had tried some other patterns like `**index.rst`, `**/doc/index.rst`, `**/*.rst`, and
```
*/
*/doc/
*/doc/index.rst
```
and
```
*/doc/index.rst
*/doc
*
```
and some other patterns I've tried days ago, which I can't remember now.

Reply:
I have done a quick try and this should work:

```rst
.. toctree::
    :glob:

    **/index
```

Reply:
It does work! I'll update the patch soon.